docs/guide/protocols/absolutesolvation.rst

```diff
@@ -1,4 +1,113 @@
 Absolute Solvation Protocol
 ===========================
 
-issue #743
\ No newline at end of file
+Overview
+--------
+
+The :class:`absolute solvation protocol <.AbsoluteSolvationProtocol>` calculates the free energy change 
+calculates the free energy change 
+associate with transferring a molecule from vacuum into a solvent.
+
+.. note::
+   Currently, water is the only supported solvent, however, more solvents might be possible in the future.
+
+The absolute solvation free energy is calculated through a thermodynamic cycle. 
+In this cycle, the interactions of the molecule are decoupled, meaning turned off, using a partial annhilation scheme (see below) both in the solvent and in the vacuum phases.
+The absolute solvation free energy is then obtained via summation of free energy differences along the thermodynamic cycle.
+
+.. figure:: img/ahfe_thermocycle.png
+   :scale: 100%
+
+   Thermodynamic cycle for the absolute solvation free energy protocol.
+
+Scientific Details
+------------------
+
+Partial annhilation scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the :class:`.AbsoluteSolvationProtocol` the coulombic interactions of the molecule are fully turned off (annihilated),
+while the Lennard-Jones interactions are decoupled, meaning the intermolecular interactions turned off, keeping the intramolecular Lennard-Jones interactions.
+
+The lambda schedule
+~~~~~~~~~~~~~~~~~~~
+
+Molecular interactions are turned off during an alchemical path using a discrete set of lambda windows. The electrostatic interactions are turned off first, followed by the decoupling of the van-der-Waals interactions. A soft-core potential is applied to the Lennard-Jones potential to avoid instablilites in intermediate lambda windows. 
+
+.. _Simulation details:
```

Comment:
I can't remember if these are glocal, if they are we might need to be careful about naming.